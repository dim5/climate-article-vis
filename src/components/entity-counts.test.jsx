import { render } from '@testing-library/react';
import './__mocks__/resize-observer.mock.js';

import EntityCounts from './entity-counts';

const data = ['AUSTRALIA', 'CALIFORNIA'];

it('should render', async () => {
  const { container } = render(<EntityCounts entities={data} />);
  expect(container).toMatchSnapshot();
});
