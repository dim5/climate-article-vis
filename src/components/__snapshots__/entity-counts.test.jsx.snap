// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div>
    <canvas
      height="0"
      style="display: block; box-sizing: border-box; width: 0px; height: 0px;"
      width="0"
    />
  </div>
</div>
`;
